include/mata/nfa/nfa.hh

```diff
@@ -356,14 +404,6 @@ OnTheFlyAlphabet create_alphabet(const std::vector<Nfa*>& nfas);
  */
 OnTheFlyAlphabet create_alphabet(const std::vector<const Nfa*>& nfas);
 
-/**
- * Check whether is the language of the automaton empty.
- * @param[in] aut Automaton to check.
- * @param[out] cex Counter-example path for a case the language is not empty.
- * @return True if the language is empty, false otherwise.
- */
-bool is_lang_empty(const Nfa& aut, Run* cex = nullptr);
-
 Nfa uni(const Nfa &lhs, const Nfa &rhs);
```

Comment:
onion

Reply:
the annoyance this generates is really high

Reply:
`nfa_union()`, as for `set_union()` in `OrdVector`?

Reply:
lang_union !

Reply:
I agree that `lang_union()` might be even better than `nfa_union()`, if we are to have some prefix to `union`, that is.